mm2src/coins/utxo.rs

```diff
@@ -1495,12 +1515,15 @@ impl UtxoHDAccount {
             .derive(account_child)
             .map_to_mm(Bip44DerPathError::from)?;
         let extended_pubkey = Secp256k1ExtendedPublicKey::from_str(&account_info.account_xpub)?;
+        let capacity =
+            account_info.external_addresses_number + account_info.internal_addresses_number + DEFAULT_GAP_LIMIT;
         Ok(UtxoHDAccount {
             account_id: account_info.account_id,
             extended_pubkey,
             account_derivation_path,
             external_addresses_number: account_info.external_addresses_number,
             internal_addresses_number: account_info.internal_addresses_number,
+            derived_addresses: HDAddressesCache::with_capacity(capacity as usize),
```

Comment:
A question for the future: will it make sense to derive addresses and fill the cache here immediately? Maybe even store these persistently in storage?

Reply:
Thank you for the question! The addresses will be derived on [EnableCoinBalanceOps::enable_coin_balance](https://github.com/KomodoPlatform/atomicDEX-API/blob/mm2.1/mm2src/coins/coin_balance.rs#L101) method call. It's called on [InitStandaloneCoinActivationOps:: get_activation_result](https://github.com/KomodoPlatform/atomicDEX-API/blob/mm2.1/mm2src/coins_activation/src/utxo_activation/init_utxo_standard_activation.rs#L82)
So we can consider that the addresses are derived on coin initialization.